environment.yml

```diff
@@ -36,4 +36,5 @@ dependencies:
   - pytest-xdist
   # docs
   - pip:
+    - git+https://github.com/openforcefield/openff-interchange@fix-1324
```

Comment:
Note that this is a branch of Interchange.

Reply:
Thanks for the reminder, I'm inclined to wait until that's released to merge this (provided the tests here still pass). We'll see how the timelines line up